### PR TITLE
review of the `docs\scos\dev\best-practices` section

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Review of the `docs\scos\dev\best-practices` section
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-tax-glue-api.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ composer require spryker/product-tax-sets-rest-api:"^2.0.0" --update-with-depend
 
 Make sure that the following module has been installed:
 
-| Module | Expected Directory |
+| MODULE | EXPECTED DIRECTORY
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-tax-glue-api.md:23]:
```suggestion
| MODULE | EXPECTED DIRECTORY |
```

File: _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-agent-assist-glue-api.md [hunk 2]
```diff
@@ -155,15 +141,15 @@ class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependency
...
+Ensure the following:
 
-*   Invalid agent access tokens passed in the `X-Agent-Authorization` header are rejected with the 401 error code.
+* Invalid agent access tokens passed in the `X-Agent-Authorization` header are rejected with the 401 error code.
 
-*   The `X-Agent-Authorization` header is required for `/agent-customer-impersonation-access-tokens` and `/agent-customer-search` resources. Requests without the header return the 401 error code.
+* The `X-Agent-Authorization` header is required for `/agent-customer-impersonation-access-tokens` and `/agent-customer-search` resources. Requests without the header return the 401 error code.
 
-*   When a valid agent access token is provided via the `X-Agent-Authorization`, the `RestRequest` object in Glue controllers populates `restUser.idAgent` with the current agent identifier.
+* When a valid agent access token is provided through the `X-Agent-Authorization`, the `RestRequest` object in Glue controllers populates `restUser.idAgent` with the current agent identifier.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-agent-assist-glue-api.md:150]:
```suggestion
* When a valid agent access token is provided through `X-Agent-Authorization`, the `RestRequest` object in Glue controllers populates `restUser.idAgent` with the current agent identifier.
```

File: _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md [hunk 3]
```diff
@@ -178,17 +173,17 @@ class UrlsRestApiDependencyProvider extends SprykerUrlsRestApiDependencyProvider
 
 {% info_block warningBox "Verification" %}
 
-Make sure that the `cms-pages` resource is available by sending the request:`GET https://glue.mysprykershop.com/cms-pages`.
-
-Make sure that the `content-product-abstract-lists` resource is available by sending the request: `GET https://glue.mysprykershop.com/content-product-abstract-lists/{% raw %}{{{% endraw %}abstract_product_list_key{% raw %}}}{% endraw %}`.
+Ensure the following:
+* The `cms-pages` resource is available by sending the request:`GET https://glue.mysprykershop.com/cms-pages`.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md:177]:
```suggestion
* The `cms-pages` resource is available by sending the request: `GET https://glue.mysprykershop.com/cms-pages`.
```

File: docs/scos/dev/best-practices/search-best-practices/precise-search-by-super-attributes.md [hunk 4]
```diff
@@ -36,81 +37,80 @@ related:
 
 ## Task to achieve
 
-Imagine shop selling laptop (abstract product) with Ram: *16/32Gb* and CPU: *i5/i7* options. One day there're only 2 models in stock: **16Gb + i5** and **32Gb + i7**.
+Imagine a shop selling a laptop (abstract product) with 16/32Gb RAM and i5/i7 CPU options. One day there are only TWO models in stock: *16Gb + i5* and *32Gb + i7*.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/best-practices/search-best-practices/precise-search-by-super-attributes.md:40]:
```suggestion
Imagine a shop selling a laptop (abstract product) with 16/32Gb RAM and i5/i7 CPU options. One day there are only two models in stock: *16Gb + i5* and *32Gb + i7*.
```

File: docs/scos/dev/best-practices/search-best-practices/precise-search-by-super-attributes.md [hunk 5]
```diff
@@ -36,81 +37,80 @@ related:
...
 
-Search is made in 2 steps:
-1. perform search  by super attributes in the **product concrete index** and get matching product abstract Ids
-2. then retrieve those from the main index for product abstracts.
+Search is done in two steps:
+1. In the **product concrete index**, search is performed by super attributes and matching product abstract IDs are displayed.
+2. Those from the main index for product abstracts are retrieved.
 
-#### Implementation
+#### Implementation plan
```
Comment (andriitserkovnyi) [inline docs/scos/dev/best-practices/search-best-practices/precise-search-by-super-attributes.md:59]: plan?

Comment (VadymSachenko) [reply docs/scos/dev/best-practices/search-best-practices/precise-search-by-super-attributes.md:59]: fixed

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**